src/mscorlib/src/System/ThrowHelper.cs

```diff
@@ -227,6 +248,94 @@ internal static class ThrowHelper {
                     argumentName = "sourceBytesToCopy";
                     break;
 
+                case ExceptionArgument.keys:
+                    argumentName = "keys";
+                    break;
+
+                case ExceptionArgument.elementType:
+                    argumentName = "elementType";
+                    break;
+
+                case ExceptionArgument.newSize:
+                    argumentName = "newSize";
+                    break;
+
+                case ExceptionArgument.len:
```

Comment:
Looks like already a PR for `Enum.ToString()` https://github.com/dotnet/coreclr/pull/3849